src/restic/node.go

```diff
@@ -227,10 +249,6 @@ func (node Node) createFileAt(path string, repo Repository) error {
 }
 
 func (node Node) createSymlinkAt(path string) error {
-	// Windows does not allow non-admins to create soft links.
```

Comment:
I think you accidentally removed this check for windows here, could you please put it back in?